src/js/block-visibility/preview.js

```diff
@@ -25,29 +29,39 @@ import './editor.scss';
  * is open to all or has visibility settings enabled.
  *
  * @since 1.1.0
- * @since 1.6.0 Use camelCase `className` in favor of `class`.
  *
  * @return {Class}
  */
 export default class Preview extends Component {
 
+	/**
+	 * Render component
+	 *
+	 * @since 1.1.0
+	 * @since 1.6.0 Use camelCase `className` in favor of `class`.
+	 * @since [version] Improve the information displayed for a restricted block.
+	 *
+	 * @return {Fragment}
```

Comment:
is this still a `{Fragment}`?

Reply:
I don't know what the real docblock for a react component html return is. I've been using `{Fragment}` not to say we're returning a `<Fragment>` but to say we're returning a "fragment of component html". The docblocking is very inconsistent across here and I don't have ESLint setup because I haven't been able to stomach taking the time to figure out how to introduce consistent JS standards across all projects. Leave this alone. We'll fix it later.